pkg/processor/worker/allocator_test.go

```diff
@@ -154,7 +233,7 @@ func benchmarkParallelAllocation(b *testing.B, numberOfWorkers int) {
 	}
 
 	// Create a new SyncPoolAllocator
-	fpa, _ := eventprocessor.NewSyncPoolAllocator(logger, eventProcessors)
+	fpa, _ := allocatorConstructor(logger, eventProcessors)
```

Comment:
What's `fpa`?
```suggestion
	allocator, _ := allocatorConstructor(logger, eventProcessors)
```

Reply:
it was for fixed pool allocator I guess, but will rename